.github/workflows/ci.yaml

```diff
@@ -53,7 +53,8 @@ jobs:
           staticcheck ./...
 
       - name: Check YAML
-        uses: ibiqlik/action-yamllint@v3
+        run: |
+          yamllint .
```

Comment:
I had to change this because the action was reporting an error, while running `yamllint` locally did not. As it turns out, Github Actions runners have yamllint installed by default, so we don't need to use a third-party action.